Directory.Build.props

```diff
@@ -8,45 +8,50 @@
     <TargetFrameworks>$(LegacyTargetFrameworks)</TargetFrameworks>
     <Platforms>AnyCPU</Platforms>
     <TargetPlatform>AnyCPU</TargetPlatform>
-    <PackageOutputPath>$(MSBuildThisFileDirectory)\nuget</PackageOutputPath>
-    <DocumentationFileValue>$(MSBuildThisFileDirectory)\bin\$(MSBuildProjectName)\$(Configuration)\$(TargetFramework)\$(MSBuildProjectName).xml</DocumentationFileValue>    
+    <ErrorReport>prompt</ErrorReport>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <WarningsAsErrors />
+    <CheckForOverflowUnderflow>true</CheckForOverflowUnderflow>
+    <AllowUnsafeBlocks>false</AllowUnsafeBlocks>
   </PropertyGroup>
   <PropertyGroup>
```

Comment:
you might want to label this group and the one above also

Reply:
I was not able to come up with nice names for now :(

Reply:
Updated, thanks

Reply:
Perfect.